README.md

```diff
@@ -77,11 +83,33 @@ docker-compose exec phpfpm composer apply-coding-standards
 Check markdown docs:
 
 ```sh
-yarn markdownlint README.md docs
+yarn coding-standards-check/markdownlint
```

Comment:
This should only be `yarn coding-standards-check` (similar to `composer check-coding-standards`). And your should be consistent on if it's called “check-coding-standards” or “coding-standards-check”.